### PR TITLE
Submit button forms

### LINKED ISSUE #2166: data download buttons are broken
Buttons no longer work:

image

### PR DESCRIPTION
**Pull request by christianmemije**
Fixes #2166

### CODE REVIEW

File: kolibri/plugins/management/assets/src/views/data-page/index.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
         {{$tr('detailsSubHeading')}}
       </p>
       <form :action="sessionlogurl" method="get">
```
Comment (jonboiser) [inline kolibri/plugins/management/assets/src/views/data-page/index.vue]: Maybe change these to `@submit`, this is the only place in the code where `action` attribute is used.

Comment (christianmemije) [reply kolibri/plugins/management/assets/src/views/data-page/index.vue]: @jonboiser Addressed in the latest commit.

File: kolibri/plugins/coach/assets/src/state/actions/exam.js [hunk 2]
```diff
@@ -291,16 +291,12 @@ function deleteExam(store, examId) {
   );
 }
 
-function getAllExercisesWithinTopic(store, channelId, topicId) {
+function getAllExercisesWithinTopic(store, topicId) {
```
Comment (jonboiser) [inline kolibri/plugins/coach/assets/src/state/actions/exam.js]: What are the changes in this file for?

Comment (christianmemije) [reply kolibri/plugins/coach/assets/src/state/actions/exam.js]: :rage4: Brb.

Comment (christianmemije) [reply kolibri/plugins/coach/assets/src/state/actions/exam.js]: @jonboiser Commit removed!

**Pull request merged**